tests/storage/test_txn_limit.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2014-2021 The Matrix.org Foundation C.I.C.
```

Comment:
we don't ask you to assign copyright, so this is yours:

```suggestion
# Copyright 2021 Toni Spets
```

Reply:
It's fine 😄